scripts/build/gen_isr_tables.py

```diff
@@ -64,37 +72,25 @@ def read_intlist(elfobj, syms, snames):
         const void *param;
     };
     """
-    intList_sect = None
     intlist = {}
-
     prefix = endian_prefix()
 
+    # Extract header and the rest of the data
     intlist_header_fmt = prefix + "II"
+    header_sz = struct.calcsize(intlist_header_fmt)
+    header_raw = struct.unpack_from(intlist_header_fmt, intlist_data, 0)
+    debug(str(header_raw))
```

Comment:
As the rebuild is big, I was trying to leave some logical steps in the commits. But I also think we could merge this two commits:
- scripts: gen_isr_tables: Code cleanup 
- scripts: gen_isr_tables: Break code into classes 

If only you think it would be more convenient.